### PR TITLE
Data hub release notes for 2.29.0

### PR DESCRIPTION
**Pull request by elize-vdr**

### CODE REVIEW

File: content/en/docs/releasenotes/data-hub/_index.md [hunk 1]
```diff
@@ -11,6 +11,12 @@ These release notes cover changes made to the [Mendix Data Hub](/data-hub/).
 
 ## 2022
 
+### April 14th, 2022
+
+#### New Feature
+
+* Deletion of Applications through the Registration API are now possible
```
Comment (reinouts) [inline content/en/docs/releasenotes/data-hub/_index.md]: Perhaps we should link to https://docs.mendix.com/apidocs-mxsdk/apidocs/data-hub-apis/#registration here?

Comment (johannahemminger) [reply content/en/docs/releasenotes/data-hub/_index.md]: @reinouts sure thing, I will add that link as well as the link to the API itself (https://datahub-spec.s3.eu-central-1.amazonaws.com/index.html).

Comment (elize-vdr) [reply content/en/docs/releasenotes/data-hub/_index.md]: Please hang on to this, I now see there are some stories missed, they were not in the same Release JIRA, adding them now

Comment (elize-vdr) [reply content/en/docs/releasenotes/data-hub/_index.md]: OK, done, not sure if one says "an MxDock notification", "a MxDock notification" or "In-app Notification" on the second feature entry

File: content/en/docs/releasenotes/data-hub/_index.md [hunk 2]
```diff
@@ -7,10 +7,21 @@ weight: 30
...
+These release notes cover changes made to the [Mendix Data Hub](/data-hub/). 
 
 ## 2022
 
+### April 14th, 2022
+
+#### New Feature
+
+* You can now delete applications through the **Registration API**. Check out the [Data Hub Registration API](https://datahub-spec.s3.eu-central-1.amazonaws.com/registration_v4.html), or read the documentation in the [Registration API](/apidocs-mxsdk/apidocs/data-hub-apis/#registration/) section of *Data Hub APIs*.
+* When a user is assigned to an Application as the Technical Business Owner, then an MxDock notification is sent to the user to inform the user of this fact.
```
Comment (Adam-Dupaski) [inline content/en/docs/releasenotes/data-hub/_index.md]: We don't use "MxDock" in docs: https://docs.mendix.com/developerportal/

We also don't have any documentation on notifications, but that really shouldn't be necessary anyway.

File: content/en/docs/releasenotes/data-hub/_index.md (hunk 1 above)
Comment (johannahemminger) [reply content/en/docs/releasenotes/data-hub/_index.md]: No worries, I'll take a look and sort it. Thanks a lot!

**Review by johannahemminger — APPROVED** [review]

**Pull request merged**